src/db/sysdb_ops.c

```diff
@@ -3409,24 +3399,12 @@ int sysdb_store_custom(struct sss_domain_info *domain,
 
     for (i = 0; i < attrs->num; i++) {
         msg->elements[i] = attrs->a[i];
-        if (add_object) {
-            msg->elements[i].flags = LDB_FLAG_MOD_ADD;
-        } else {
-            el = ldb_msg_find_element(resp[0], attrs->a[i].name);
-            if (el == NULL) {
-                msg->elements[i].flags = LDB_FLAG_MOD_ADD;
-            } else {
-                msg->elements[i].flags = LDB_FLAG_MOD_REPLACE;
-            }
-        }
+        msg->elements[i].flags = LDB_FLAG_MOD_ADD;
+        msg->elements[i].flags = LDB_FLAG_MOD_ADD;
```

Comment:
It's just a copy & paste error, right? There's no reason for setting twice the flag here.